src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -64,6 +67,43 @@ export class ManifoldSubscriptionList {
     this.getSubscriptions(this.owner);
   }
 
+  ctaHref(planID: string) {
+    if (!this.baseUrl || this.baseUrl === '#') {
+      return this.baseUrl;
+    }
+
+    // const search = new URLSearchParams();
+    // set plan ID
+    // search.set('planId', planID);
+
+    // // set configurable feature selection (or skip, if no configurable features);
+    // Object.entries(this.userSelection[planID] || {}).forEach(([key, val]) => {
+    //   search.set(key, `${val}`);
+    // });
+
+    return `${this.baseUrl}/${planID.toString()}`;
+  }
+
+  handleCtaClick = (planId: string) => (e: MouseEvent) => {
+    e.preventDefault();
+    this.ctaClick.emit({ id: `manifold-cta-plan-${planId}` });
+
+    // this.connection;
+    // .analytics
+    // .track({
+    //   description: 'Track pricing matrix cta clicks',
+    //   name: 'click',
+    //   type: 'component-analytics',
+    //   properties: {
+    //     planId,
+    //   },
+    // })
+    // .finally(() => {
+    const anchor = e.srcElement as HTMLAnchorElement; // ?
+    window.location.href = anchor.href; // ?
```

Comment:
Is this part of the analytics tracker or do we need it here? Do we need the analytics tracker? It seemed to not be on the connection object.

Reply:
Hmm. `analytics` should definitely be on the connection object though. And yeah I think we're going to need it tracked but we can check with Jeff to be sure.